### PR TITLE
Fixes: small modifications and added SMS channel

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: channels/sms.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+Enter these in the Zammad admin interface at SMS accounts. 
+
+.. figure:: /images/channels/sms/sms_channel_settings.png
+   :alt: Channel settings SMS
+   :align: center
+
+
+Further options are to enter a sender number and selecting a destination group.
```
Comment (dvuckovic) [inline channels/sms.rst:30]: `selecting` => `select`

File: channels/sms.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   :alt: Notification settings
+   :align: center
+
+You can either save your configuration directly, or let your settings be 
+checked by running a test in advance.
+
+Now it's possible to send SMS notifications via trigger or automation. Setup
+is analogue to setting up email notifications. Please note that user objects
+belonging to the person you want to notify must contain a mobile phone number as
+an attribute. If there is no phone number registered, no SMS is sent.
```
Comment (dvuckovic) [inline channels/sms.rst:64]: `no SMS is sent` => `no SMS will be sent`

**Review by dvuckovic — APPROVED** [review]
@ralf401 Great job! Just a couple of minor suggestions for you consideration.

File: channels/sms.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Communication via SMS is available directly from inside Zammad. 
+
+Notifications via SMS
+---------------------
+
+It's also possible to receive notifications via SMS. These are messages sent by
+trigger or automation. Setup is done in the admin interface at Channels > SMS
+
+As service providers, Massenversand, message_bird and twilio are supported.
```
[the remark this answers is not in the document]
Comment (ralf401) [reply channels/sms.rst:49]: see first comment

File: channels/sms.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+.. figure:: /images/channels/sms/sms_notification.png
+   :alt: Notification settings
+   :align: center
+
+You can either save your configuration directly, or let your settings be 
+checked by running a test in advance.
+
+Now it's possible to send SMS notifications via trigger or automation. Setup
+is analogue to setting up email notifications. Please note that user objects
```
[the remark this answers is not in the document]
Comment (ralf401) [reply channels/sms.rst:62]: Done. Added an additional "the" => [...] that the user object belonging [...]

File: channels/sms.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+SMS integration primarily lets you create tickets, add articles and respond to
+tickets via SMS. It is also possible to send notifications, e.g. to agents.
+
+You can configure SMS notifications in the channels section in the admin
+interface at Channels -> SMS. 
+
+Configure incoming SMS
+----------------------
+
+To process incoming SMS, message_bird and twilio are currently the only
```
[the remark this answers is not in the document]
Comment (dvuckovic) [reply channels/sms.rst:13]: I'd say so, the issue here is in the GUI: it should show proper names not these "vars" :)

[the remark this answers is not in the document]
Comment (dvuckovic) [reply channels/sms.rst:13]: Feel free to open a follow-up issue for GUI asking for this change. It is obviously missing, considering the third option is properly named.

**Pull request merged**